.travis.yml

```diff
@@ -26,8 +26,8 @@ deploy:
   on:
     tags: true
     all_branches: true
-    rvm: 1.9.3
-    rvm: 2.0
```

Comment:
I am ok with this as it is an EOL version, I would just like to know what failed that made you want to remove this.